freqtrade/optimize/backtesting.py

```diff
@@ -399,6 +402,16 @@ def _get_adjust_trade_entry_for_candle(self, trade: LocalTrade, row: Tuple
                 self.wallets.update()
                 return pos_trade
 
+        if stake_amount is not None and stake_amount < 0.0:
+            amount = -stake_amount / current_rate
+            if amount > trade.amount:
+                logger.info(f"Amount is higher than available. {amount} > {trade.amount}")
```

Comment:
Please abstain from logging in backtesting.

it'll be meaningless log messages (you're testing 2000 trades, and will not know what you tested with), and will slow down hyperopt (output is a deal breaker there, as the terminal is usually the slowest part).